### PR TITLE
fix: missed translation in ar

### PR DESCRIPTION
**Pull request by ahmedsamirdev**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Added missing translation for the Arabic language |
| 🇨🇳 Chinese | 为阿拉伯语添加了缺失的翻译 |

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
图片

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ tag: vVERSION
 - Major version release is not included in this schedule for breaking change and new features.
 
 ---
+- Internationalization*
+💄 Added missing translation for the Arabic language.
```
Comment (afc163) [inline CHANGELOG.en-US.md:19]: Please leave changelog in PR's body.

**Comment by ahmedsamirdev** [comment]
- Internationalization*
💄 Added missing translation for the Arabic language.

**Comment by li-jia-nan** [comment]
Hi, There are some keys you should create Pull Request to `rc-picker` instead of `ant-design`

**Comment by li-jia-nan** [comment]
Please update snap:

```bash
npm run test:update components/locale
```

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**